src/types/transaction.rs

```diff
@@ -57,6 +57,8 @@ pub struct Receipt {
   pub contract_address: Option<H160>,
   /// Logs generated within this transaction.
   pub logs: Vec<Log>,
+  /// Status: either 1 (success) or 0 (failure).
+  pub status: Option<u32>,
```

Comment:
Since the spec says `QUANTITY` it should be encoded as hex. To achieve that we need to use a type from `ethereum_types`. I think the smallest there is `U64` so we should use this.

Corresponding code from Parity:
https://github.com/paritytech/parity/blob/afae75b57cd70d81e968fc66319431871a79ff3d/rpc/src/v1/types/receipt.rs#L53